general-preprocessing-rainfall_noaa.ipynb

```diff
@@ -8,11 +8,11 @@
     "\n",
```

Comment:
<p>For accessibility purposes it is better to label links with descriptive text. So, rather than "obtained from this landing page" you could say "obtained from the NOAA Physical Sciences Laboratory", or similar. </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Environmental-DS-Book/general-preprocessing-rainfall_noaa/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Thanks and I've revised based on your suggestion, please check latest version on GitHub (review_round1 branch).</p>